docs/cluster_up_down.md

```diff
@@ -2,9 +2,13 @@
 
 - [Overview](#overview)
 - [Getting Started](#getting-started)
-  - [Mac OS X](#mac-os-x)
-  - [Windows](#windows)
   - [Linux](#linux)
+  - [Mac OS with Docker for Mac](#mac-os-with-docker-for-mac)
```

Comment:
The latest version of Mac OS dropped the "X" ... and Docker for Mac only supports this latest version.
docker-machine can be run on OS X